packages/server/utils/OpenAIServerManager.ts

```diff
@@ -78,18 +78,15 @@ class OpenAIServerManager {
             content: prompt
           }
         ],
-        temperature: 0.7,
-        max_tokens: 80,
-        top_p: 1,
```

Comment:
According to OpenAI [doc](https://platform.openai.com/docs/api-reference/chat/create), best practice is to not set **both** `temperature` and `top_p`